src/main/java/no/fint/betaling/controller/MeController.java

```diff
@@ -109,11 +107,15 @@ private String getName(PersonResource person) {
                 : String.format("%s %s %s", n.getFornavn(), n.getMellomnavn(), n.getEtternavn());
     }
 
-    private User getUserFromSkoleressursByFeidenavn(String ePPN) {
+    private User getUserFromSkoleressursByFeidenavn(String ePPN) throws PersonNotFoundException {
         User user = new User();
-
-        SkoleressursResource skoleressurs = restUtil.get(SkoleressursResource.class,
-                UriComponentsBuilder.fromUriString(schoolResourceEndpoint).pathSegment("feidenavn", ePPN).build().toUriString());
+        SkoleressursResource skoleressurs;
+        try {
+            skoleressurs = restUtil.get(SkoleressursResource.class,
+                    UriComponentsBuilder.fromUriString(schoolResourceEndpoint).pathSegment("feidenavn", ePPN).build().toUriString());
+        } catch (RuntimeException ex) {
+            throw new PersonNotFoundException(ex.getMessage());
+        }
```

Comment:
You should catch `HttpStatusCodeException` instead - then you can validate that the REST status code is in fact `404` and rethrow if it's not.

Reply:
I will try, had an issue where the catch didnt apply when I tried earlier. Will test again and reply

Reply:
@asgeirn would this be sufficient?